sources/iam/policy_test.go

```diff
@@ -373,6 +373,9 @@ func TestNewPolicySource(t *testing.T) {
 		defer span.End()
 
 		items, err := source.List(ctx, sources.FormatScope(account, ""), false)
+		if len(items) == 0 {
```

Comment:
This was causing a panic in my local setup.
Maybe it is related to my permissions.

Reply:
@DavidS-ovm 
I've reverted my changes in that test.

I've also reverted all the changes related to ARN.

This PR only adds 2 new resources and updates the `ItemMapper` as @dylanratcliffe suggested.

I run the tests locally and they are passing except this test, which I believe something to do with my permissions.
I see that the test in CI are also failing, and they seem to be related to my permissions as well.

Let me know if there is any change you want me to do on the PR.